app/src/androidTest/java/com/appunite/loudius/LoginScreenTest.kt

```diff
@@ -67,12 +56,6 @@ class LoginScreenTest {
     @JvmField
     val screenshotTestRule = ScreenshotTestRule()
 
-    @Before
-    fun setUp() {
-        hiltRule.inject()
-    }
-
-    @BindValue
```

Comment:
bug: You've removed `BindValue` from tests, this means that `GithubHelperModule` is not replaced with this implementation. This definitely should break our UI tests, but somehow they are not executed. This should be also investigated why UI tests are not executed.

Reply:
thanks for noticing !

Reply:
or actually from what I understand this was kind of removing the GitHelperModule which provided GithubHelper so that we could mock it, but now since we are not using Hilt for injection, we do not need to do it

Reply:
nevermind I see what you mean now, trying to fix it ...